go/vt/vtgate/engine/fake_vcursor_test.go

```diff
@@ -991,7 +991,7 @@ func printBindVars(bindvars map[string]*querypb.BindVariable) string {
 			fmt.Fprintf(buf, " ")
 		}
 
-		if bindvars[k].Type == querypb.Type_TUPLE {
+		if bindvars[k].Type == querypb.Type_ROW_TUPLE {
 			fmt.Fprintf(buf, "%s: [", k)
```

Comment:
do we need this for both the types? TUPLE and ROW_TUPLE

Reply:
no